dist/images/ovnkube.sh

```diff
@@ -0,0 +1,266 @@
+  echo ${ovn_host}
+  date
+  if [[ ${ovn_master} = "true" ]]
+  then
+    echo "==================== ovnkube-master =================== "
+    echo "====================== ovnnb_db.pid"
+    if [[ -f /var/run/openvswitch/ovnnb_db.pid ]]
+    then
+      cat /var/run/openvswitch/ovnnb_db.pid
+    fi
+    echo "====================== ovnsb_db.pid"
+    if [[ -f /var/run/openvswitch/ovnsb_db.pid ]]
+    then
+      cat /var/run/openvswitch/ovnsb_db.pid
+    fi
+    echo "====================== ovs-vswitchd.pid"
+    if [[ -f /var/run/openvswitch/ovs-vswitchd.pid ]]
+    then
+      cat /var/run/openvswitch/ovs-vswitchd.pid
+    fi
+    echo "====================== ovsdb-server.pid"
+    if [[ -f /var/run/openvswitch/ovsdb-server.pid ]]
+    then
+      cat /var/run/openvswitch/ovsdb-server.pid
+    fi
+    echo "====================== ovsdb-server-nb.log"
+    if [[ -f /var/log/openvswitch/ovsdb-server-nb.log ]]
+    then
+      cat /var/log/openvswitch/ovsdb-server-nb.log
+    fi
+    echo "====================== ovsdb-server-sb.log "
+    if [[ -f /var/log/openvswitch/ovsdb-server-sb.log ]]
+    then
+      cat /var/log/openvswitch/ovsdb-server-sb.log
+    fi
+    echo "====================== ovn-northd.pid"
+    if [[ -f /var/run/openvswitch/ovn-northd.pid ]]
+    then
+      cat /var/run/openvswitch/ovn-northd.pid
+    fi
+    echo " "
+    echo "====================== ovn-northd.log"
+    if [[ -f /var/log/openvswitch/ovn-northd.log ]]
+    then
+      cat /var/log/openvswitch/ovn-northd.log
+    fi
+    echo " "
+    echo "====================== ovnkube-master.pid"
+    if [[ -f /var/run/openvswitch/ovnkube-master.pid ]]
+    then
+      cat /var/run/openvswitch/ovnkube-master.pid
+    fi
+    echo " "
+    echo "====================== ovnkube-master.log"
+    if [[ -f /var/log/openvswitch/ovnkube-master.log ]]
+    then
+      cat /var/log/openvswitch/ovnkube-master.log
+    fi
+  fi
+  echo " "
+  echo "==================== ovnkube =================== "
+  echo "====================== ovn-controller.pid"
+  if [[ -f /var/run/openvswitch/ovn-controller.pid ]]
+  then
+    cat /var/run/openvswitch/ovn-controller.pid
+  fi
+  echo " "
+  echo "====================== ovn-controller.log"
+  if [[ -f /var/log/openvswitch/ovn-controller.log ]]
+  then
+    cat /var/log/openvswitch/ovn-controller.log
+  fi
+  echo " "
+  echo "====================== ovnkube.pid"
+  if [[ -f /var/run/openvswitch/ovnkube.pid ]]
+  then
+    cat /var/run/openvswitch/ovnkube.pid
+  fi
+  echo " "
+  echo "====================== ovnkube.log"
+  if [[ -f /var/log/openvswitch/ovnkube.log ]]
+  then
+    cat /var/log/openvswitch/ovnkube.log
+  fi
+  echo " "
+  echo "====================== ovn-k8s-cni-overlay.log"
+  if [[ -f /var/log/openvswitch/ovn-k8s-cni-overlay.log ]]
+  then
+    cat /var/log/openvswitch/ovn-k8s-cni-overlay.log
+  fi
+}
+
+setup_cni () {
+  # Take over network functions on the node
+  rm -Rf /etc/cni/net.d/*
+  rm -Rf /host/opt/cni/bin/*
```

Comment:
Let's not remove everything here; let's just remove the ovn-k8s-cni-overlay binary.

Reply:
Do we still 
# rm -Rf /etc/cni/net.d/*